docs: add known issue PE-7218 (#7563)
spectrocloud/librarium: docs/docs-content/release-notes/known-issues.md

> | For locally deployed clusters, adding a custom `stylus.path` to the `user-data` file causes cluster creation to fail as it causes the Palette agent to be unable to find the `user-data` file.                                                                                                                                                                                                                                                                                                                                                                                                                      | No workaround available.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | July 28, 2025      | Edge                         |

Reviewer: ```suggestion
| For locally deployed clusters, adding a custom `stylus.path` to the `user-data` file causes cluster creation to fail, as it prevents the Palette agent from locating the `user-data` file.                                                                                                                                                                                                                                                                                                                                                                                                    | No workaround available.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | July 28, 2025      | Edge                         |
```

Reviewer: I also have a question about the date (July 28, 2025). I previously got the information that we backdated known issues to match the closest release. And there is no release today, as far as I know.
